src/webextension/locales/en-US/list.ftl

```diff
@@ -46,10 +46,10 @@ toolbar-item-count =
 
 toolbar-add-item = New Entry
 toolbar-go-home = Home
-toolbar-send-feedback = Provide Feedback
+toolbar-send-feedback = Submit Feedback
```

Comment:
>"Submit Feedback" really sounds like the thing that happens after you've filled out the feedback form and are sending it to us, especially since "Submit" is a somewhat-technical term

@changecourse I presume "Submit" was an intentional decision made recently?

Reply:
This was a copy request from our team, but I tend to agree with you... I'm not happy about "Submit" as opposed to "Provide"

@sandysage final call.

Reply:
Let's stick with "provide" as @changecourse suggests.